### DISCUSSION TITLE
Does addon support exchange server traffic?

### DISCUSSION THREAD

**Discussion by PrimaryInc** [hassio-addons/addon-nginx-proxy-manager#237] [Q&A]
Hi,

Exchange 2019 server runing internally..
I know it a bit overkill for a home, but I use the exchange server for jobs related activities as well as privat emails.
Use case, I have a few applications runing, more than one using (80)/443 and i use a synology nas reverse proxy to be able to access apps by hostname externally. Biggest issue I have is that synology does not support proxying exchange server and client traffic.

Question is if ngix addon for HA support it?
If so, how? :)

Thank you!!

**Comment by frenck** [comment]
I have no idea, sorry

**Reply by PrimaryInc** [reply]
That was quick, thanks :) Guess I have to try and see.
